pages/common/yt-dlp.md

```diff
@@ -25,14 +25,14 @@
 
 `yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
 
-- Download all playlists of YouTube channel/user keeping each playlist in separate directory:
+- Download all playlists of a YouTube channel/user keeping each playlist in a separate directory:
 
-`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/user/TheLinuxFoundation/playlists}}"`
+`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
 
-- Download Udemy course keeping each chapter in separate directory under MyVideos directory in your home:
+- Download a Udemy course keeping each chapter in a separate directory:
 
-`yt-dlp -u {{user}} -p {{password}} -P "{{~/MyVideos}}" -o "{{%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s}}" "{{https://www.udemy.com/java-tutorial}}"`
+`yt-dlp -u {{user}} -p {{password}} -P "{{path/to/directory}}" -o "{{%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s}}" "{{https://www.udemy.com/java-tutorial}}"`
 
-- Download entire series season keeping each series and each season in separate directory under C:/MyVideos:
+- Download the entire series season keeping each series and each season in a separate directory:
 
-`yt-dlp -P "{{C:/MyVideos}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{https://videomore.ru/kino_v_detalayah/5_sezon/367617}}"`
+`yt-dlp -P "{{path/to/directory}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```

Comment:
```suggestion
`yt-dlp -P "{{path/to/directory}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{https://videomore.ru/kino_v_detalayah/5_sezon/367617}}"`
```

Reply:
Is this URL valid because I got 403?

Reply:
> Is this URL valid because I got 403?

Oh, I didn't check for it (just reverted the change), do you have any suggestions for a new one?

Reply:
> > Is this URL valid because I got 403?
> 
> Oh, I didn't check for it (just reverted the change), do you have any suggestions for a new one?

No, because I don't understand this example fully.

Reply:
> Maybe a playlist like [youtube.com/playlist?list=PLqE7PUanPIFTs1Zy1Lz270nUHhtZcx66e&si=VTWFdggY2cbLCLsP](https://youtube.com/playlist?list=PLqE7PUanPIFTs1Zy1Lz270nUHhtZcx66e&si=VTWFdggY2cbLCLsP)?

Using the command with this playlist resulted in a folder like this:
```
└── NA
    └── NA - NA
        ├── NA - NA.mkv
        └── NA - NA.webm
```
Maybe just remove the example?

Reply:
@kbdharun what you think about deleting this example?

Reply:
> @kbdharun what you think about deleting this example?

I guess it's fine to delete it, as it seems to be a bit complicated.